src/SDKs/ManagementGroups/Management.ManagementGroups/Properties/AssemblyInfo.cs

```diff
@@ -10,8 +10,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Management Groups Library")]
 [assembly: AssemblyDescription("Provides functionality to handle management groups.")]
 
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("1.2.0.0")]
+[assembly: AssemblyFileVersion("1.2.0.0")]
```

Comment:
Only update the `AssemblyFileVersion` here

Reply:
Okay. Thanks. Will make the change.